libdokan/tlf.go

```diff
@@ -157,13 +157,14 @@ func (tlf *TLF) GetFileInformation(ctx context.Context, fi *dokan.FileInfo) (st
 }
 
 // open tries to open a file.
-func (tlf *TLF) open(ctx context.Context, oc *openContext, path []string) (dokan.File, bool, error) {
+func (tlf *TLF) open(ctx context.Context, oc *openContext, path []string) (
+	dokan.File, dokan.CreateStatus, error) {
 	if len(path) == 0 {
 		if err := oc.ReturningDirAllowed(); err != nil {
-			return nil, true, err
+			return nil, 0, err
```

Comment:
Is this behavior correct? It used to return `true`, which should map to `dokan.ExistingDir`. Was that previously a bug?

Reply:
Here CreateFile fails so the CreateStatus is ignored.

Reply:
SGTM